<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Presidios-MacBook-Pro-4.local" failures="1" tests="4" name="tests.ProductSortTest" time="14.403" errors="0" timestamp="2025-04-01T11:44:50 IST" skipped="3">
  <testcase classname="tests.ProductSortTest" name="testSortByPriceLowToHigh" time="3.374">
    <failure message="failed" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: failed
at org.testng.AssertJUnit.fail(AssertJUnit.java:65)
at tests.ProductSortTest.testSortByPriceLowToHigh(ProductSortTest.java:49)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- testSortByPriceLowToHigh -->
  <testcase classname="tests.ProductSortTest" name="testSortByPriceLowToHigh" time="3.265">
    <skipped/>
  </testcase> <!-- testSortByPriceLowToHigh -->
  <testcase classname="tests.ProductSortTest" name="testSortByPriceLowToHigh" time="3.835">
    <skipped/>
  </testcase> <!-- testSortByPriceLowToHigh -->
  <testcase classname="tests.ProductSortTest" name="testSortByPriceLowToHigh" time="3.929">
    <skipped/>
  </testcase> <!-- testSortByPriceLowToHigh -->
</testsuite> <!-- tests.ProductSortTest -->
